### PR TITLE
Adding Modal with information about search operators

### PR DESCRIPTION
**Pull request by MaciejWiczk**
Code taken from https://www.positronx.io/create-react-modal-popup-with-material-ui/, hopefully it's in proper format. Docs are the same as in RTFD https://rfhub2.readthedocs.io/en/latest/source/searching.html#with-search-bar

### CODE REVIEW

File: frontend/src/components/SearchModal.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { makeStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import Modal from '@material-ui/core/Modal';
+
+function rand() {
+    return Math.round(Math.random() * 20) - 10;
+}
+
+function getModalStyle() {
```
Comment (pbylicki) [inline frontend/src/components/SearchModal.tsx:9]: why is this randomized?

**Comment by pbylicki** [comment]
@MaciejWiczk  Can you provide some screenshot?

File: frontend/src/components/SearchModal.tsx (hunk 1 above)
Comment (MaciejWiczk) [reply frontend/src/components/SearchModal.tsx:9]: oh there is no point, removed!

**Comment by MaciejWiczk** [comment]
> @MaciejWiczk Can you provide some screenshot?

nObDw26qAl

**Comment by pbylicki** [comment]
@MaciejWiczk What do you think about changing "how to search" text into a help outline icon?
https://material-ui.com/components/material-icons/

`import HelpOutlineIcon from '@material-ui/icons/HelpOutline';`

Apart from that, the change looks good :)

**Comment by MaciejWiczk** [comment]
> @MaciejWiczk What do you think about changing "how to search" text into a help outline icon?
> https://material-ui.com/components/material-icons/
> 
> `import HelpOutlineIcon from '@material-ui/icons/HelpOutline';`
> 
> Apart from that, the change looks good :)

Looks much more pro, I will add it tomorrow!:)

**Comment by MaciejWiczk** [comment]
@pbylicki here are changes to IconButton with Icon:
vRAzM4y9Ga

**Review by MarcinMaciaszek — APPROVED** [review]

**Review by pbylicki — APPROVED** [review]

**Pull request merged**